### PR TITLE
add: `parsec-deb`

### LINKED ISSUE #2145: PacReq: parsec
### Types

-deb (Debian achieve release of the package)

### URL

https://parsec.app/downloads

### Building guide

_No response_

### Additional context

Hello,

It will be greate if we have parsec app some day 😄.

### PR DESCRIPTION
**Pull request by Henryws**
Fixes #2145

### CODE REVIEW

File: packages/parsec-deb/parsec-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name="parsec-deb"
+gives="parsec"
+version="150-86e"
+url="https://builds.parsec.app/package/parsec-linux.deb"
+description="Simple, low-latency game streaming"
+hash="c30072f3a4ddac7687f626b8d32876e327791bce164d74448a147ef48ea8eae5"
```
Comment (wizard-28) [inline packages/parsec-deb/parsec-deb.pacscript:6]: As the `url` is a dynamic link, the `hash` would be automatically invalidated in the future when a new version releases. Consider using a static `url` instead (an `url` where you can plug the `version` in), or remove the `hash`.

---

Add `maintainer`.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/parsec-deb/parsec-deb.pacscript (hunk 1 above)
Comment (Henryws) [reply packages/parsec-deb/parsec-deb.pacscript:6]: They don't offer versioned URLs. I'm also not going to maintain this.

Comment (wizard-28) [reply packages/parsec-deb/parsec-deb.pacscript:6]:
```suggestion
version="150-86e"
url="https://builds.parsec.app/package/parsec-linux.deb"
description="Simple, low-latency game streaming"
```

Remove `hash` then

**Review by wizard-28 — APPROVED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add incompatible

**Comment by wizard-28** [comment]
It's also incompatible with `ubuntu:rolling`, and `ubuntu:devel`

**Pull request merged**